Sources/SKCore/BuildSetup.swift

```diff
@@ -0,0 +1,39 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2019 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Foundation
+import Utility
+import SKSupport
+
+/// Build configuration
+public struct BuildSetup {
+
+  /// Default configuration
+  public static let `default` = BuildSetup(configuration: .debug,
```

Comment:
I'm curious why you chose a static property for this instead of just making this the default initializer?

Reply:
verbosity and explicitly I guess.